src/AddTodoForm.js

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 
-function AddTodoForm(props){
+function AddTodoForm({ onAddTodo }) {
+    const [todoTitle, setTodoTitle] = React.useState('');
+
+    const handleTitleChange = (event) => {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle)
+    }
     const handleAddTodo = (event) => {
         event.preventDefault();
-        const todoTitle = event.target.title.value
-        console.log(todoTitle)
-        event.target.title.value = '';
-        props.onAddTodo(todoTitle)
+        const uniqueId = Date.now()
+        const newObj = {
+            title: todoTitle,
+            id: uniqueId,
+        }
+        onAddTodo(newObj);
+        setTodoTitle('');
     };
-    return(
+    return (
         <form onSubmit={handleAddTodo}>
             <label htmlFor='todoTitle'>Title:</label>
-            <input id='todoTitle' type='text' name='title'/>
+            <input id='todoTitle' value={todoTitle} type='text' name='title' onChange={handleTitleChange} />
```

Comment:
Let's break this up a bit. It's a long code line. Limit code lines to 80 characters. There should be a line in your editor that represents this.